.github/workflows/actions.yml

```diff
@@ -0,0 +1,82 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
i'm wondering whether we want to have builds before a PR is started on the branch in same repo?

Reply:
Sure, I can change it.